app/src/main/java/com/keylesspalace/tusky/components/timeline/viewmodel/TimelineViewModel.kt

```diff
@@ -181,16 +181,16 @@ abstract class TimelineViewModel(
 
     protected fun shouldFilterStatus(statusViewData: StatusViewData): Filter.Action {
         val status = statusViewData.asStatusOrNull()?.status ?: return Filter.Action.NONE
-        if (status.actionableStatus.account.id == account.accountId) {
```

Comment:
That was a sort of guess at that time "never filter own posts"?

And the actual change/fix here?

And the remainder is only reformatting?

Reply:
> That was a sort of guess at that time "never filter own posts"?

I made https://github.com/tuskyapp/Tusky/pull/4742 because Mastodon filters should not apply to own posts to match Mastodon behavior. I included Tusky-only filters as well, but based on user feedback I'm reverting that part.